.pre-commit-config.yaml

```diff
@@ -8,3 +8,10 @@
     hooks:
     -   id: isort
         args: ["-rc", "."]
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: 'v0.761'
+    hooks:
+    -   id: mypy
```

Comment:
[Notes]
It didn't work by `args: ["appium"]`.

https://github.com/pre-commit/mirrors-mypy/issues/5#issuecomment-515841288